docs/getting-started.md

```diff
@@ -83,6 +83,10 @@ To get a production build, run:
 npm run build:js
 ```
 
+### Live reload
+
+We don't provide it OOTB. You can setup your own solution and `DISABLE_PREVENT=1 npm run dev` will help you with unwanted `beforeunload` alert.
```

Comment:
IMHO, `=true`/`=false` are more natural.

Reply:
It's cast to bool in the code, so it's just the docs that'd need update.

Will do that in #1476